src/util/strencodings.h

```diff
@@ -29,6 +29,22 @@ enum SafeChars
     SAFE_CHARS_URI, //!< Chars allowed in URIs (RFC 3986)
 };
 
+/**
+ * Used by ParseByteUnits()
+ * SI are 1000 base while IEC is 1024 base
+ */
+enum ByteUnitMultiplier {
```

Comment:
Use enum class, which can be used like `ByteUnit::g` or `ByteUnit::K`.

Reply:
changed. more intuitive, clear.